community/detectors/xwiki_cve_2024_21650/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'java-library'
+}
+
+description = 'Tsunami Apache APISIX RCE (CVE-2022-24112) VulnDetector plugin.'
```

Comment:
Fix the description to match this detector's name

Reply:
Oops, updated